CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+- Rename `lib` directories to `utils` @ecraig12345 ([#2153](https://github.com/microsoft/fluent-ui-react/pull/2153))
```

Comment:
Not sure that this is a really breaking change as I don't see anything that can affect consumer that uses public API.

Reply:
That's good to know, thank you! Some of my experience with Fabric has led me to be very cautious about assuming internal path changes are non-breaking, but if you think it's safe here that's great.